### PR TITLE
added logical_or function to tf frontend 

### LINKED ISSUE #13683: logical_or
- [ ] #1545

### PR DESCRIPTION
**Pull request by Sooryak12**
This pull request makes the following change :
Close #13683

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/math.py [hunk 1]
```diff
@@ -188,6 +188,9 @@ def logical_and(x, y, name="LogicalAnd"):
 def logical_xor(x, y, name="LogicalXor"):
     return ivy.logical_xor(x, y)
 
+@to_ivy_arrays_and_back
+def logical_or(*, x, y, name="logical_or"):
```
[the remark this answers is not in the document]
Comment (Sooryak12) [reply ivy/functional/frontends/tensorflow/math.py]: sure.

**Review by hmahmood24 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 2]
```diff
@@ -385,6 +385,36 @@ def test_tensorflow_logical_and(
     )
 
 
+# logical_or
+@handle_frontend_test(
+    fn_tree="tensorflow.math.logical_or",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple([ivy.bool]),
```
[the remark this answers is not in the document]
Comment (Sooryak12) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py]: Hi @hmahmood24 sorry for the late replies ,I'm not receiving any notifications .first of all thanks a lot for helping me out . Even after running pre-commit formatting test case fails . what should I do ?

[the remark this answers is not in the document]
Comment (Sooryak12) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py]: @hmahmood24 I have reverted the accidental change in .idea .I hope my PR is good to be merged ! Thanks

**Review by hmahmood24 — APPROVED** [review]

**Comment by hmahmood24** [comment]
Thanks for the changes @Sooryak12!

Just a small point. Kindly ensure that your PRs get correctly linked to the issue(s) you are trying to resolve by commenting: `Close #<issue_number>` as explained [here](https://lets-unify.ai/ivy/overview/contributing/the_basics.html#todo-list-issues). This just helps in keeping track of tasks that have been carried out and correctly closing any issues that should be closed once the PR gets merged. For now, I'll just manually link the two but in the future you should do this yourself. 😊

Thanks for a great contribution! 🚀

**Pull request merged**